site/content/docs/v1.4/api-types/_index.md

```diff
@@ -0,0 +1,20 @@
+---
+version: v1.4
```

Comment:
Given this is versioned, do we need to update our `make gen-docs` script?

Reply:
see my [other comment](https://github.com/vmware-tanzu/velero/pull/2839#issuecomment-676824233), i dont think we need to make a change